### PR TITLE
First overlay's BSS symbols moved to C

### PR DESCRIPTION
**Pull request by ethteck**
This PR migrates a all of the first overlay's BSS symbols into C and out of undefined_syms. There are still BSS issues to sort out, but this brings us one step closer to shiftability,

### CODE REVIEW

**Comment by BowserSlug** [comment]
⚠️ This PR introduces a warning:
- src/26840_len_20d0.c:142: warning: assignment makes integer from pointer without a cast
- src/341d0.c:54: warning: initialization makes integer from pointer without a cast

**Comment by JoshDuMan** [comment]
My only feedback would be to put somewhere either in bss.c or the splat splits why that files exists. Even if its removed later, its best to not let it slip that its a compiler hack to force correct bss order.

File: src/341d0.c [hunk 1]
```diff
@@ -37,20 +37,20 @@ s32 D_8007F1D0[] = {
...
-s32 D_8007F200[] = {
+s32 D_8007F200[1] = {
     D_8007F1D0
 };
 
-s32 D_8007F204[] = {
-    D_8007F1D0, 0x00000000, 0x00000000
+s32 D_8007F204[1] = {
+    D_8007F1D0
 };
```
Comment (nanaian) [inline src/341d0.c:54]: Do these even need to be arrays at this point?

**Review by nanaian — COMMENTED** [review]
👀

**Review by Brotenko — APPROVED** [review]

**Pull request merged**